index.html

```diff
@@ -231,7 +232,7 @@ <h2 class="about_title">About me</h2>
             Get in touch
           </button>
         </form>
-       </section>
+    </section>
       <section class="footer_section">
```

Comment:
Kindly makes sure your icon design stick as much as possible to the Figma design provided as is shown in the picture below the color is not the same. To fix that I can suggest you use this  `color: #505F79` is the one used in the Figma design :+1: 

## You design 

![image](https://user-images.githubusercontent.com/95297251/194140177-53a71aff-afaa-408a-96e8-0ee5af221457.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/194141866-4d48f15d-f216-42d4-8282-03f101fee100.png)